### DISCUSSION TITLE
Message: "Web2PrintToolsBundle\Controller\AdminController" has no container set, did you forget to define it as a service subscriber?

### DISCUSSION THREAD

**Discussion by ismetsener** [pimcore/pimcore#9121] [Q&A / Support]
Hi,

In Pimcore-10  I get this error when I opened:
Favorite Output Channels Configuration

"Web2PrintToolsBundle\Controller\AdminController" has no container set, did you forget to define it as a service subscriber?
Server threw exception - could not perform action. Please reload the admin interface and try again.

Timestamp: Fri May 14 2021 17:20:26 GMT+0200 (Midden-Europese zomertijd)
Status: 500 | Internal Server Error
URL: /admin/web2printtools/admin/favorite-output-definitions-table-proxy?xaction=read&_dc=1621005625968
Method: POST
Message: "Web2PrintToolsBundle\Controller\AdminController" has no container set, did you forget to define it as a service subscriber?
Trace: 
in /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerResolver.php:39
#0 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerResolver.php(29): Symfony\Bundle\FrameworkBundle\Controller\ControllerResolver->configureController()
#1 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php(120): Symfony\Bundle\FrameworkBundle\Controller\ControllerResolver->instantiateController()
#2 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php(42): Symfony\Component\HttpKernel\Controller\ControllerResolver->createController()
#3 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php(86): Symfony\Component\HttpKernel\Controller\ContainerControllerResolver->createController()
#4 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/TraceableControllerResolver.php(38): Symfony\Component\HttpKernel\Controller\ControllerResolver->getController()
#5 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(140): Symfony\Component\HttpKernel\Controller\TraceableControllerResolver->getController()
#6 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(79): Symfony\Component\HttpKernel\HttpKernel->handleRaw()
#7 /opt/pimcore-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(195): Symfony\Component\HttpKernel\HttpKernel->handle()
#8 /opt/pimcore-demo/public/index.php(36): Symfony\Component\HttpKernel\Kernel->handle()
#9 {main}

**Comment by fashxp** [comment]
please create an issue here https://github.com/pimcore/web2print-tools ... or even better a PR to fix it ;-)

**Reply by ismetsener** [reply]
thanks, I did :)

**Reply by ismetsener** [reply]
I created the issue in  pimcore/web2print-tools
how can I close this issue here?

even better a PR to fix it ;-)
I will try, I have experience in the IT, but not as developer, I am reading a lot and learn day by day
